### DISCUSSION TITLE
Build errors : openid-client

### DISCUSSION THREAD

**Discussion by carlevans72** [nextauthjs/next-auth#4153] [Help]
### Question 💬

Hi,

I've been using v4.29 fine and all of a sudden today I started getting the following error during build:

/home/vsts/work/1/s/node_modules/openid-client/lib/client.js
Module parse failed: Unexpected character '#' (168:2)

I notice that openid-client released a new version 5 days ago and was wondering whether this was a known issue.

I didn't want to create a bug report if it was something particular to my setup.

Thanks,
Carl.

### How to reproduce ☕️

/home/vsts/work/1/s/node_modules/openid-client/lib/client.js
Module parse failed: Unexpected character '#' (168:2)

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by ThangHuuVu** [comment]
Do you mean you're using v3.29?
If you think there's a problem with `openid-client` and you are using a lock file, try to enforce the package version by `npm install --no-save openid-client@<WORKING_VERSION>`

**Reply by carlevans72** [reply]
Sorry - late night typing!

I'm using 4.2.1 and getting that error.

Thanks again,
Carl

**Reply by ThangHuuVu** [reply]
I did a quick test and was not able to reproduce this error. This might be related to the Node.js version. See: https://github.com/panva/node-openid-client/discussions/461
If the above doesn't solve the problem, could you add a minimal reproduction and share here? :pray:
